metalkube/kni-installer
# Merge openshift/installer 4.2.0-0.ci-2019-05-29-154816

To test:
- Check out this PR to ~/go/src/github.com/openshift-metalkube/kni-installer
- Use https://github.com/openshift-metal3/dev-scripts/pull/577
- Run `KNI_INSTALL_FROM_GIT=true make`

Will update the dev-scripts PR with an actual KNI image once this is merged.

See @russellb's comment below, worker deployments don't currently work.
